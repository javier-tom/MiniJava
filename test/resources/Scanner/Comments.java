// Single line comment
//* Also a single line comment

System.out.println(five);

/* Multi line comment
int five = 5;
 */

System.out.println(two);

// int five = 5;