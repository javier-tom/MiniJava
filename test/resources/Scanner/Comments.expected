SOUT LPAREN ID(five) RPAREN SEMICOLON SOUT LPAREN ID(two) RPAREN SEMICOLON